UI-MVC/Models/ProjectViewModel.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+using Domain.Projects;
+using Domain.Users;
+
+namespace UIMVC.Models
+{
+    public class ProjectViewModel
+    {
+        public Project Project { get; set; }
```

Comment:
zoals besproken hier een currentphase erbij of ProjectViewModel.Project.Currentphase vullen.